packages/flags/src/index.ts

```diff
@@ -1,5 +1,5 @@
 export const FEATURES = {
-  walletLockUnlock: true,
```

Comment:
I had good intentions to add wallet unlock when I originally created this 😂 I'll come back to this